test/common/upstream/cluster_manager_impl_test.cc

```diff
@@ -232,22 +232,6 @@ TEST_F(ClusterManagerImplTest, UnknownHcType) {
   EXPECT_THROW(create(parseBootstrapFromJson(json)), EnvoyException);
 }
 
-TEST_F(ClusterManagerImplTest, MaxClusterName) {
-  const std::string json = R"EOF(
-  {
-    "clusters": [
-    {
-      "name": "clusterwithareallyreallylongnamemorethanmaxcharsallowedbyschema"
-    }]
-  }
-  )EOF";
-
-  EXPECT_THROW_WITH_MESSAGE(create(parseBootstrapFromJson(json)), Json::Exception,
```

Comment:
Isn't this test still valid? Or is it not anymore?

Reply:
Same comment as above. Test just moved to utility_test.cc ..

Reply:
Yup sounds good